src/main/scala/info/bethard/timenorm/formal/Readers.scala

```diff
@@ -136,7 +136,7 @@ class AnaforaReader(val DCT: Interval)(implicit data: Data) {
       case ("Two-Digit-Year", Some(value), N, N, N, None) => value.partition(_ != '?') match {
         case (year, questionMarks) => YearSuffix(interval(properties), year.toInt, questionMarks.length)
       }
-      case ("Between", None, N, N, N, None) => Between(interval(properties, "Start-"), interval(properties, "End-"))
+      case ("Between", None, N, N, N, None) => Between(interval(properties, "Start-"), interval(properties, "End-"), properties.get("Start-Included"), properties.get("End-Included"))
```

Comment:
Do the conversion to Boolean here, in the Reader. (In general, conversions from Strings to other types belong in Reader.) Something like:
```
properties.get("Start-Included") match {
    case Some("Included") => true
    case Some("NotIncluded") => false
    case None => false
}
```